ControlRoomApplication/ControlRoomApplication/Controllers/RadioTelescopeControllers/RadioTelescopeController.cs

```diff
@@ -1116,5 +1120,17 @@ private void CheckAndRunSoftwareStops()
                 }
             }
         }
+        /// <summary>
+        /// This is the method that checks for acceptable discrepancy in the absolute and motor encoders, returning a bool (true if within acceptable range)
+        /// </summary>
+        public bool CompareMotorAndAbsoluteEncoders (Orientation motor, Orientation absolute)
+        {
+            // Compare discrepancy of current orientations and keep below constant
+            if (Math.Abs(motor.Elevation - absolute.Elevation) <= Constants.MiscellaneousConstants.MOTOR_ABSOLUTE_ENCODER_DISCREPANCY && 
+                Math.Abs(motor.Azimuth - absolute.Azimuth) <= Constants.MiscellaneousConstants.MOTOR_ABSOLUTE_ENCODER_DISCREPANCY)
```

Comment:
Just something to keep in mind for the future - The elevation absolute encoder does not have the full range of motion that we need (I think it can only read 92 degrees but its supposed to read 100). Depending on final changes, we may have to account for the "dead zone" that the encoder will not be able to read.